source/Nuke.Common/Tools/SqlPackage/SqlPackage.json

```diff
@@ -0,0 +1,3145 @@
+              },
+              {
+                "name": "AllowDropBlockingAssemblies",
+                "type": "bool",
+                "help": "This property is used by SqlClr deployment to cause any blocking assemblies to be dropped as part of the deployment plan. By default, any blocking/referencing assemblies will block an assembly update if the referencing assembly needs to be dropped."
+              },
+              {
+                "name": "AllowExternalLanguagePaths",
+                "type": "bool",
+                "help": "Allows file paths, if available, to be used to generate external language statements."
+              },
+              {
+                "name": "AllowExternalLibraryPaths",
+                "type": "bool",
+                "help": "Allows file paths, if available, to be used to generate external library statements."
+              },
+              {
+                "name": "AllowIncompatiblePlatform",
+                "type": "bool",
+                "help": "Specifies whether to attempt the action despite incompatible SQL Server platforms."
+              },
+              {
+                "name": "AllowUnsafeRowLevelSecurityDataMovement",
+                "type": "bool",
+                "help": "Do not block data motion on a table that has Row Level Security if this property is set to true. Default is false."
+              },
+              {
+                "name": "AzureSharedAccessSignatureToken",
+                "type": "string",
+                "help": "Azure shared access signature (SAS) token. See [SqlPackage for Azure Synapse Analytics](sqlpackage-for-azure-synapse-analytics.md#publish-import-data)."
+              },
+              {
+                "name": "AzureStorageBlobEndpoint",
+                "type": "string",
+                "help": "Azure Blob Storage endpoint, see [SqlPackage for Azure Synapse Analytics](sqlpackage-for-azure-synapse-analytics.md#publish-import-data)."
+              },
+              {
+                "name": "AzureStorageContainer",
+                "type": "string",
+                "help": "Azure Blob Storage container, see [SqlPackage for Azure Synapse Analytics](sqlpackage-for-azure-synapse-analytics.md#publish-import-data)."
+              },
+              {
+                "name": "AzureStorageKey",
+                "type": "string",
+                "help": "Azure storage account key, see [SqlPackage for Azure Synapse Analytics](sqlpackage-for-azure-synapse-analytics.md#publish-import-data)."
+              },
+              {
+                "name": "AzureStorageRootPath",
+                "type": "string",
+                "help": "Storage root path within the container. Without this property, the path defaults to `servername/databasename/timestamp/`. See [SqlPackage for Azure Synapse Analytics](sqlpackage-for-azure-synapse-analytics.md#publish-import-data)."
```

Comment:
Markdown inline code is not supported, should be using `<c>`. Same goes for a couple of other properties, that can be considered as code (and which aren't even using markdown inline code).